medusa/server/api/v2/config.py

```diff
@@ -131,7 +131,8 @@ def get(self, query=''):
                 }
             },
             'selectedRootIndex': int(app.SELECTED_ROOT) if app.SELECTED_ROOT else None,
-            'backlogPeriod': app.BACKLOG_PERIOD
+            'backlogPeriod': app.BACKLOG_PERIOD,
```

Comment:
Would it not be better to use something like this to keep the backlog stuff together?
```
'backlog': {
    'period': app.BACKLOG_PERIOD,
    'status': app. BACKLOG_STATUS
}